src/product.html

```diff
@@ -15,12 +15,12 @@
         <!-- endbower -->
         <link rel="stylesheet" href="css/main.css">
     </head>
-    <body>
+    <body ng-app="data-fetch">
```

Comment:
As we discussed, `data-fetch` isn't really the best name for this... `TIY-Catalog` is the name of the project, so if you need a module that acts as a _part_ of that project `TIY-Catalog--` is a good prefix to start with. This is `product.html`, which should present a single `listing` record, so maybe `TIY-Catalog--Listing` is the appropriate name?